src/components/scenes/CryptoExchangeScene.tsx

```diff
@@ -179,25 +172,6 @@ export class CryptoExchangeComponent extends React.Component<Props, State> {
     if (this.timeoutId != null) clearTimeout(this.timeoutId)
   }
 
-  static getDerivedStateFromProps(props: Props, state: State) {
-    if (props.forceUpdateGuiCounter !== state.forceUpdateGuiCounter) {
-      return {
-        fromAmountNative: mul(props.fromExchangeAmount, props.fromWalletPrimaryInfo.exchangeDenomination.multiplier),
-        fromExchangeAmount: props.fromExchangeAmount,
-        toExchangeAmount: props.toExchangeAmount,
-        forceUpdateGuiCounter: props.forceUpdateGuiCounter
-      }
-    } else {
-      // Check which wallet we are currently editing.
-      // Only change the exchangeAmount of the opposite wallet to prevent feedback loops
-      if (state.whichWalletFocus === 'from') {
-        return { toExchangeAmount: props.toExchangeAmount }
-      } else {
-        return { fromExchangeAmount: props.fromExchangeAmount }
-      }
```

Comment:
Deleting this `else` logic worries me. It doesn't seem related to the max send feature, but to general flip input instability.

Reply:
Good catch, reverted that part